.github/workflows/integration.yaml

```diff
@@ -38,25 +38,22 @@ jobs:
           pip install tox
       - name: Run test
         run: tox -e unit
-        #  Integration Tests won't be executed for now as there are still some
-        #  issues with pytest-operator and k8s charms.
-        #
-        #  integration-test:
-        #    name: Integration tests
-        #    runs-on: ubuntu-latest
-        #    steps:
-        #      - name: Check out code
-        #        uses: actions/checkout@v2
-        #      - name: Setup Python
-        #        uses: actions/setup-python@v2
-        #        with:
-        #          python-version: 3.8
-        #      - name: Install Dependencies
-        #        run: |
-        #          pip install tox
-        #      - name: Setup operator environment
-        #        uses: charmed-kubernetes/actions-operator@master
-        #        with:
-        #            provider: microk8s
-        #      - name: Run test
-        #        run: tox -e integration -- -m container
+  integration-test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install Dependencies
+        run: |
+          pip install tox
+      - name: Setup operator environment
+        uses: charmed-kubernetes/actions-operator@master
+        with:
+            provider: microk8s
+      - name: Run test
+        run: sg microk8s -c "tox -e integration"
```

Comment:
I don't think this `sg` is necessary.  I [don't bother with it in `kube-state-metrics-operator`](https://github.com/johnsca/kube-state-metrics-operator/blob/main/.github/workflows/tests.yaml#L40), and it works fine.  If there is something that requires this, I'd prefer to add something to `actions-operator` to resolve it (such as changing socket file permissions) so that we don't have to worry about doing this anywhere.